### DISCUSSION TITLE
The rewards are decreasing every day

### DISCUSSION THREAD

**Discussion by yahai8** [Lilypad-Tech/lilypad#343] [Q&A]
Since I joined the LilyPad network, the "sepolia.arbiscan" website shows that there are transactions every day, but my rewards are decreasing every day.this is my address：0x24E84aD0CAd3a3c3BB978544ff9eD226773281b4

**Comment by robertjitaru** [comment]
@yahai8, when did you initially connect the node to the network? The number of nodes increases from day to day and then the reward is distributed to a larger pool.

**Reply by galaxyxtwo** [reply]
@yahai8 it looks like youre now earning Lilybits consistently! Rewards earned per day will vary each day. You can check your total rewards at https://info.lilypad.tech and see points earned per day at https://rp-points.lilypad.tech.
